src/icons/TagIcon.tsx

```diff
@@ -30,7 +30,9 @@ export const TagIcon = forwardRef<
       <defs>
         <clipPath id={`${ids[0]}`}>
           <path
-            fill={secondaryColor ? `${theme[secondaryColor]}` : '#fff'}
+            fill={
+              secondaryColor ? `${theme[secondaryColor]}` : `${theme.gray300}`
+            }
```

Comment:
This is not duotone, but the color logic of 2 colors icon is applied. Would you check if default `gray300` is applicable for this icon's secondary color, because the previous default color for this icon is `white` (#fff)?

Reply:
Good catch on this. I rewrote the logic for `getColorVariables`. Please see that file for my changes. Thanks Thien!